src/api/deposit/batchExchangeAddresses.ts

```diff
@@ -1,15 +1,11 @@
 import { Network } from 'types'
-import networks from '@gnosis.pm/dex-contracts/networks.json'
 
-function _getAddress(networkId: number): string {
-  const networkInfo = networks['BatchExchange'][networkId]
-  return networkInfo ? networkInfo.address : ''
-}
+import { getGpV1ContractAddress } from 'utils/contract'
 
 const addressesByNetwork = {
-  [Network.Rinkeby]: _getAddress(Network.Rinkeby),
-  [Network.Mainnet]: _getAddress(Network.Mainnet),
-  [Network.xDAI]: _getAddress(Network.xDAI),
+  [Network.Rinkeby]: getGpV1ContractAddress(Network.Rinkeby) as string,
```

Comment:
why is the `as` required? can you copy the return value of the previous function?

Reply:
That's because it returns `null` and the original function returns only `string`.
Did not want to change for legacy reasons (update gp-v1 code).

What I can do is to - instead of casting to string - return an empty string: `getGpV1ContractAddress(Network.Rinkeby) || ''`

Will update in another PR

Reply:
https://github.com/gnosis/gp-ui/pull/279